pilot/pkg/model/jwks_resolver.go

```diff
@@ -78,6 +79,15 @@ var (
 
 	// Close channel
 	closeChan = make(chan bool)
+
+	networkFetchSuccessCounter = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
small nit: let's call these `pilot_jwks_resolver_network_fetch_success_total` and `pilot_jwks_resolver_network_fetch_fail_total` to match the "best practices" from: https://prometheus.io/docs/practices/naming/

Reply:
Thanks!